### ISSUE TITLE
question

### ISSUE THREAD

**Issue by DenceChen** [huseinzol05/Stock-Prediction-Models#1]
hi i find some problem, like (14.lstm-seq2seq-attention.ipynb), the output **self.logits** it has shape  shape=(1, 6),but self.Y has the shape shape=(5, 6),are you sure this? think you ans.

**Comment by huseinzol05** [comment]
Ya, not realized that, thanks! should be fixed for next push.
